app-development/charter.md

```diff
@@ -25,6 +25,16 @@ Yet users expect the UI to be as polished and smooth as media playback, with gra
 
 The browser contains other rendering and computing solutions which are near native speeds and can give developers more control and flexibility over device limits and UI janks. For instance, frameworks that leverage WebAssembly for computing and canvas-based technologies such as WebGL and WebGPU for rendering, could be used to create an optimized web runtime for developing UIs targeted at TV sets and set-top boxes.
 
+### API support for TV specific features
+
+Standards need to be created to allow browsers on TVs to expose TV-focused APIs, similar to how mobile-focused web APIs have been created to support the capabilities of mobile devices, such as touch events, Sensor APIs (accelerometer, gyroscope, etc.), the Geolocation API, and more.
+
+Some examples of APIs that would be useful include:
+
+* Video playback layer support - Some devices always playback video on the lowest layer making it very difficult to create experiences with an embedded video on top of a UI
```

Comment:
I don't understand what that means in practice and how that describes something different from other devices. Could this point be clarified?

Browsers typically render video on the lowest layer when they do hardware decoding. Is this more pointing at the fact that TV apps are rendered on top of a video pane on TV devices and that TV apps need an API to control that pane, e.g. something like the [TV Control API](https://w3c.github.io/tvcontrol-api/)?

Reply:
For the purpose of the charter, could we make this more generic and not necessarily cite specific examples? The detail would be for the TF to gather requirements for API features

Reply:
Video Playback is something we should research. On RDK OS - if you playback a video in an HTML page, the video player is zIndex=0 and you can't change it. For Lightning which uses a canvas tag this adds a layer of complexity needing to cut a hole in the canvas tag. Then if you want to move / resize video player you need to move video tag and coordinate with a "hole punch". Ideally the video tag should support zIndex and just show up on top. I'm not sure how it is on other platforms.